auth/session.go

```diff
@@ -67,6 +69,13 @@ func (auth *Authenticator) AuthenticateCookie(rq *http.Request, response http.Re
 	user, err := auth.GetUser(session.Username)
 	if user != nil && user.Disabled() {
 		user = nil
+		return user, err
+	}
+	if err != nil {
+		return nil, err
+	}
+	if !bytes.Equal(session.SessionUUID, user.GetSessionUUID()) {
+		return nil, base.HTTPErrorf(http.StatusUnauthorized, "User has changed since session acquired")
```

Comment:
I think a better message might be "Session no longer valid for user".

Reply:
fixed.